kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/serialization/debug/SerializationDebugRequestHandler.java

```diff
@@ -150,18 +151,18 @@ public boolean handleRequest(VaadinSession vaadinSession,
 
     static class Runner implements Consumer<HttpServletRequest> {
 
-        private final Consumer<Result> onSuccess;
+        private final Consumer<Result> onComplete;
         private final int serializationTimeout;
 
-        public Runner(Consumer<Result> onSuccess, int serializationTimeout) {
-            this.onSuccess = onSuccess;
+        public Runner(Consumer<Result> onComplete, int serializationTimeout) {
+            this.onComplete = onComplete;
             this.serializationTimeout = serializationTimeout;
         }
 
-        private void executeOnSuccess(Result result) {
-            if (onSuccess != null) {
+        private void executeOnComplete(Result result) {
+            if (onComplete != null) {
                 try {
-                    onSuccess.accept(result);
+                    onComplete.accept(result);
                 } catch (Exception ex) {
                     // Do not interrupt the request
                     ex.printStackTrace();
```

Comment:
This is probably a left over from initial implementation. Could it be replaced by a logger?
Maybe accessing `LoggerFactory` directly here instead of capturing instances, to prevent unwanted serialization issue.